TrackPropagation/RungeKutta/test/rkTest_cfg.py

```diff
@@ -25,10 +25,12 @@
 
 #process.load("Configuration.StandardSequences.MagneticField_35T_cff")
 
-process.load("Configuration.StandardSequences.MagneticField_38T_cff")
+#process.load("Configuration.StandardSequences.MagneticField_38T_cff")
 
 #process.load("Configuration.StandardSequences.MagneticField_40T_cff")
 
+process.load("MagneticField.Engine.volumeBasedMagneticField_090322_2pi_scaled_cfi")
```

Comment:
and by the answer by Nicola, I understand that what I do here is fully supported
" you don't care about what map you get and want to avoid specifying a GT in a Q&D .py."